<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagerDialog</class>
 <widget class="QDialog" name="UserManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnCreate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbRole">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsername">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
